project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,206 @@
+          - or:
+            - Clothes Dryer|Electric
+            - Clothes Dryer|Electric, Premium
+      - &energystar_clothes_dryer_gas 
+        option: Clothes Dryer|Gas, Premium, EnergyStar
+        apply_logic:
+          - or:
+            - Clothes Dryer|Gas
+            - Clothes Dryer|Gas, Premium
+
+  - upgrade_name: All ENERGYSTAR Most Efficient Appliances
+    options:
+      - &energystar_most_eff_refrigerator
+        option: Refrigerator|EF 15.9 #TBD
+        apply_logic:
+          - or:
+            - Refrigerator|EF 6.7
+            - Refrigerator|EF 10.2
+            - Refrigerator|EF 10.5
+        costs:
+          - value: 2500.0 #TBD
+            multiplier: Fixed (1)
+        lifetime: 17.4
+      - &energystar_most_eff_clothes_washer
+        option: Clothes Washer|EnergyStar Most Efficient
+        apply_logic:
+        - or:
+            - Clothes Washer|Standard
+            - Clothes Washer|EnergyStar
+            - Clothes Washer|EnergyStar, Cold Only
+            - Clothes Washer|EnergyStar More Efficient
+      - &energystar_most_eff_dishwasher 
+        option: Dishwasher|240 Rated kWh
+        apply_logic:
+          - or:
+            - Dishwasher|318 Rated kWh
+            - Dishwasher|290 Rated kWh
+            - Dishwasher|270 Rated kWh
+            - Dishwasher|255 Rated kWh
+      - &energystar_most_eff_clothes_dryer_electric 
+        option: Clothes Dryer|Electric, Premium, EnergyStar Most Efficient
+        apply_logic:
+          - or:
+            - Clothes Dryer|Electric
+            - Clothes Dryer|Electric, Premium
+            - Clothes Dryer|Electric, Premium, EnergyStar
+      - &energystar_most_eff_clothes_dryer_gas 
+        option: Clothes Dryer|Gas, Premium, EnergyStar Most Efficient
+        apply_logic:
+          - or:
+            - Clothes Dryer|Gas
+            - Clothes Dryer|Gas, Premium
+            - Clothes Dryer|Gas, Premium, EnergyStar
+
+  - upgrade_name: ENERGYSTAR Electric Dryer
+    options:
+      - &energystar_electric_dryer
+        option: Clothes Dryer|Electric, Premium, EnergyStar
+        apply_logic:
+          - or:
+            - Clothes Dryer|Gas
+            - Clothes Dryer|Gas, Premium
+            - Clothes Dryer|Propane
+
+  - upgrade_name: Induction Range & Oven
+    options:
+      - &induction_range_oven 
+        option: Cooking Range|Electric Induction
+        apply_logic:
+          - or:
+            - Cooking Range|Gas
+            - Cooking Range|Propane
+
+  - upgrade_name: Conventional Range & Oven
+    options:
+      - &conventional_range_oven 
+        option: Cooking Range|Electric Conventional
+        apply_logic:
+          - or:
+            - Cooking Range|Gas
+            - Cooking Range|Propane
+
+  - upgrade_name: Electric Pool Heaters
+    options:
+      - &electric_pool_heaters 
+        option: Misc Pool Heater|Electricity
+        apply_logic:
+          - Misc Pool Heater|Natural Gas
+      - &electric_pool_heaters_78F 
```

Comment:
This and the next two option do not exist in project_national, can remove

Reply:
Done